<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GOMainWindow</class>
 <widget class="QMainWindow" name="GOMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>GO Methodology</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionNew_Markov"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionClose_C"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menuAnalysis_A">
    <property name="title">
     <string>Analysis(&amp;A)</string>
    </property>
    <addaction name="actionAnalysis_Probability"/>
    <addaction name="actionFind_Path"/>
    <addaction name="actionFind_Cut"/>
   </widget>
   <widget class="QMenu" name="menuLanguage_L">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Language(&amp;L)</string>
    </property>
    <addaction name="actionChinese_Simplified"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuView_V">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <addaction name="actionTool_DockWidget"/>
    <addaction name="actionParameter_DockWidget"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuView_V"/>
   <addaction name="menuAnalysis_A"/>
   <addaction name="menuLanguage_L"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarAnalysis">
   <property name="windowTitle">
    <string>Analysis</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/new.png</normaloff>:/icon/resource/image/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNew_Markov">
   <property name="text">
    <string>New Markov</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/open.png</normaloff>:/icon/resource/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_C">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/close.png</normaloff>:/icon/resource/image/close.png</iconset>
   </property>
   <property name="text">
    <string>Close(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/save.png</normaloff>:/icon/resource/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/saveas.png</normaloff>:/icon/resource/image/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/saveall.png</normaloff>:/icon/resource/image/saveall.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/quit.png</normaloff>:/icon/resource/image/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionChinese_Simplified">
   <property name="text">
    <string>Chinese Simplified</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/about.png</normaloff>:/icon/resource/image/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/help.png</normaloff>:/icon/resource/image/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAnalysis_Probability">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/start.png</normaloff>:/icon/resource/image/start.png</iconset>
   </property>
   <property name="text">
    <string>Analysis Probability</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionFind_Path">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/path.png</normaloff>:/icon/resource/image/path.png</iconset>
   </property>
   <property name="text">
    <string>Find Path</string>
   </property>
  </action>
  <action name="actionFind_Cut">
   <property name="icon">
    <iconset resource="../resource.qrc">
     <normaloff>:/icon/resource/image/cut.png</normaloff>:/icon/resource/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>Find Cut</string>
   </property>
  </action>
  <action name="actionTool_DockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tool DockWidget</string>
   </property>
  </action>
  <action name="actionParameter_DockWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Parameter DockWidget</string>
   </property>
  </action>
  <action name="actionFile_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>File Toolbar</string>
   </property>
  </action>
  <action name="actionAnalysis_Toolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Analysis Toolbar</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
